src/idle.jl

```diff
@@ -0,0 +1,75 @@
+function idle_callback(handle)
+    idle = @Base.handle_as handle UvWorkerIdle
+    progress(idle.worker)
```

Comment:
@vtjnash this progress function will call callbacks that need to run on the Julia thread.

Reply:
I do not recommend using uv_idle in Julia. It is preferable to use `@async` instead, which is a a much more powerful version of the same functionality.